SMPyBandits/Policies/GreedyOracle.py

```diff
@@ -0,0 +1,34 @@
+from .IndexPolicy import IndexPolicy
```

Comment:
Missing docstring for the module! Please add some documentation.

Reply:
Just a comment of one or a few lines please?

Reply:
I did put one :
`author: Julien Seznec
Reference: [Heidari et al., 2016, https://www.ijcai.org/Proceedings/16/Papers/224.pdf]
Tight Policy Regret Bounds for Improving and Decaying Bandits.
Hoda Heidari, Michael Kearns, Aaron Roth.
International Joint Conference on Artificial Intelligence (IJCAI) 2016, 1562.`
Can't you see him ?

Reply:
I would like to have a comment on the module level, just so it displays nicely in the doc.

Reply:
There is a problem with github display : it is outdated.
Check here https://github.com/SMPyBandits/SMPyBandits/pull/206/files#diff-19c0dcbe97f5820bf219e780ba4a3fb7